megatron/megatron/training/arguments.py

```diff
@@ -1631,7 +1631,8 @@ def _add_training_args(parser):
                        choices=['nccl', 'ucc'],
                        help='Select a communicator backend for pipeline parallel communication. '
                        'If None, the default backend will be used.')
-
+    group.add_argument('--enable-zero-bubble', action='store_true',
+                       help='Use Zero Bubble (ZB-H1) scheduling, which splits BW into B and W.')
```

Comment:
`enable-zero-bubble` is temporarily not compatible with features like `sequence_parallel`、`transformer-engine`, you can check this when validating the args.